uber/templates/preregistration/preregbase.html

```diff
@@ -39,7 +39,9 @@
                 onClick: function () {
                     $('.group_fields').addClass('hide');
                     $('input[name="badge_type"]').val('{{ c.ATTENDEE_BADGE }}');
```

Comment:
This is the fix for most of the problems listed in #1261 - because there was a Javascript error on the ``confirm`` form, ``togglePrices()`` was never being called.